internal/clients/diffreviewer/github/github_service.go

```diff
@@ -191,16 +194,18 @@ func (s *Service) LoadConfig(nightfallConfigFileName string) (*nightfallconfig.C
 	}
 	nightfallConfig, err := nightfallconfig.GetNightfallConfigFile(workspacePath, nightfallConfigFileName)
 	if err != nil {
-		s.Logger.Error("Error getting Nightfall config file. Ensure you have a Nightfall config file located in the root of your repository at .nightfalldlp/config.json with at least one Detector enabled")
-		return nil, err
+		s.Logger.Warning("Issue retrieving valid Nightfall config file, using default detectors instead. Ensure you have a Nightfall config file located in the root of your repository at .nightfalldlp/config.json with at least one Detector enabled")
+		nightfallConfig = &nightfallconfig.NightfallConfigFileStructure{
```

Comment:
If we want to have a default config for all the CIs we support, then I am wondering if this logic should go inside of `nightfallconfig.GetNightfallConfigFile`, thoughts?

Reply:
Good idea, that would get rid of the duplicate code